src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -165,7 +137,6 @@
   ;; for component tests only
   [{default-on-confirm       :on-confirm
     default-limit-crypto     :limit-crypto
-    default-crypto-decimals  :crypto-decimals
```

Comment:
the component tests are stubbing this value, so we no longer need the default here.